### PR TITLE
Add OpenVINO plugin RegionYolo Op

### PR DESCRIPTION
**Pull request by YangleiZouIntel**
This PR needs https://github.com/plaidml/openvino/pull/127

Algorithm follows [OpenVino RegionYolo reference inplementation](https://github.com/openvinotoolkit/openvino/blob/master/ngraph/core/reference/include/ngraph/runtime/reference/region_yolo.hpp)

More information about OpenVINO yolo region layer:  [OpenVINO Region layer](https://docs.openvinotoolkit.org/2020.1/_docs_MO_DG_prepare_model_convert_model_tf_specific_Convert_YOLO_From_Tensorflow.html)

The region layer is first introduced in [darknet region layer](https://github.com/pjreddie/darknet/blob/master/src/region_layer.c)

### CODE REVIEW

File: plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/region_yolo.cpp
Comment (tzerrell) [inline plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/region_yolo.cpp:32]: As I learned [here](https://github.com/plaidml/plaidml/pull/1644#discussion_r548315375), the linter will put each element of a list on its own line if you end the list with a trailing comma. So this might look better if you put a comma between the `))` at the end of this line (and similarly for the other `::testing:Combine` lines).

**Review by tzerrell — COMMENTED** [review]
Looks good! I have a linting comment and this will need https://github.com/plaidml/openvino/pull/127 updated & landed as well.

File: plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/region_yolo.cpp
Comment (YangleiZouIntel) [reply plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/region_yolo.cpp]: Thanks,  I have updated the vector initialization lists with a trailing comma and it works very well. 

It seems `::testing:Combine` does not support trailing comma. I added `//` for each line so the code is more readable.

**Review by tzerrell — COMMENTED** [review]
LGTM, will approve once the OV reference is updated to the landed version of the commit with RegionYOLO

**Review by tzerrell — APPROVED** [review]

**Pull request merged**
